### ISSUE TITLE
[Installation Issue] Twitch won't let you grab Lithium automatically

### ISSUE THREAD

**Issue by Ruchian** [Team-RAM/RAM#66]
The twitch client as well as other clients currently cannot install the modpack. It will error out.
The reason is because it tries to download Lithium (Forge), an alpha mod, which for some reason Twitch won't let you get through the platform. (It won't show up in Search, you need to use external methods to find it).
This issue has been going on for a couple of days, I'm not sure how long it lasts.
> https://www.curseforge.com/minecraft/mc-mods/lithium-forge

I recommend users to download the mod by hand if they are capable of grabbing the other mods.
@jellysquid3 just pinging you to inform you of this as well, since looking at your activity, I am guessing this isn't something that happened intentionally.

**Comment by Darkere** [comment]
I've just installed the pack via the Twitch launcher. Twice. Without any issues. 

The reason it doesn't show up in twitch when you try to add it manually is that the project only has alpha versions. Projects like that don't appear in twitch. They still work if added to the pack by hand.

**Comment by Ruchian** [comment]
Hm, last time I was informed of this was two days ago, which is where I went to in dept testing to figure out which mod it was it wouldn't grab. It seems that the issue was present at least diffidently present 14 hours ago.
I tested it myself just now, you're correct. It seems, what I described in the OP post is particially fixed.
I got reports from people that those who use the GDLauncher can now also grab it again. .... Hm.
Since it was up for about 4 days this problem, I expected it to last a week at least and wanted to inform people.
I suppose it is no longer an issue at the moment though. 
Thanks for testing it out yourself ;p

(search still isn't fixed though. You ~~can~~ could diffidently (at some point) find alpha mods if you had the preferred release setting on Alpha (in options). 

PS: I feel completely embarrassed now. The heck nonsense is this. Sigh.
Ask ProxyNeko if you don't believe it ;p

Edit: You know what, never mind my post. 
I don't know why but indeed Twitch at the moment won't let you find alpha mods at all. The setting doesn't matter. I reported an issue that was already fixed apparently which is enough of a bothersome input. (I apologize for pinging people and spamming their email with it.) I guess I failed hard. 
I'm not sure if Twitch is planning on returning that feature. (You'll have to take my word for it again that it was possible .I can't provide evidence which is also embarrasing.) At least the pack can be grabbed and played easily which is the important part I guess.

aaaaaa~
excuse me.
